kubernetes/opencost.yaml

```diff
@@ -21,6 +22,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: opencost
+  namespace: opencost
```

Comment:
ClusterRoles are not namespaced?

Reply:
ClusterRoles are **not** namespaced. If you put this in, delete the namespace, it's left behind. If you change the namespace and `kubectl apply` you get 
```
clusterrole.rbac.authorization.k8s.io/opencost unchanged
```